tests/js/index.js

```diff
@@ -35,10 +35,11 @@ if (isNodeProcess && process.platform === 'win32') {
 }
 
 // catching segfaults during testing can help debugging
-if (isNodeProcess) {
-    const SegfaultHandler = node_require('segfault-handler');
-    SegfaultHandler.registerHandler("crash.log");
-}
+//uncomment to enable segfault handler
```

Comment:
Why don't we want to catch segfaults by default?

Reply:
Cause segfault-handler is not working right now for us and it outputs a lot of ... by default which is not useful at all